week-4/Homework/mandatory/4-groceries.js

```diff
@@ -6,7 +6,20 @@
 let groceriesToBuy = [];
 
 let groceryList = {
-  item1: "",
-  item2: "",
-  item3: ""
+  item1: "Potatoes",
+  item2: "Orange juice ",
+  item3: "Rice"
+
 };
+
+Object.values(groceryList).forEach(e => {
+  return groceriesToBuy.push(e);
```

Comment:
This is great, good job!

Can you explain what the `return` here is doing? I think if you remove it, your code will do the same thing, so it would be great to think about what it means :)